backend/src/org/commcare/util/CommCarePlatform.java

```diff
@@ -170,4 +172,13 @@ public String getMenuDisplayStyle(String menuId) {
         }
         return commonDisplayStyle;
     }
+
+    public OfflineUserRestore getDemoUserRestore() {
+        return offlineUserRestore;
+    }
+
+    @Override
+    public void registerDemoUserRestore(OfflineUserRestore offlineUserRestore) {
```

Comment:
It seems dangerous to assume there is only one of them like this, given that the spec doesn't outline (and the parser doesn't validate) that there is only one restore to process